templates/fine-tune-llm_v2/README.ipynb

```diff
@@ -6,142 +6,341 @@
+    "#### Default Mode\n",
+    "Similar to OpenAI's finetuning experience, the `default` mode provides a minimal and efficient setup. It allows you to quickly start a finetuning job by setting just a few parameters (`model_id` and `train_path`). All other settings are optional and will be automatically selected based on dataset statistics and predefined configurations.\n",
     "\n",
-    "**Optional**: You can get a WandB API key from [WandB](https://wandb.ai/authorize) to track the fine-tuning process. If not provided, you can only track the experiments through the standard output logs.\n",
+    "#### Custom Mode\n",
+    "The `custom` mode offers more flexibility and control over the finetuning process, allowing for advanced optimizations and customizations. You need to provide more configurations to setup this mode (e.g. prompt format, hardware, batch size, etc.)\n",
     "\n",
-    "Next, you can launch a fine-tuning job with your WandB API key passed as an environment variable."
+    "Here's a comparison of the two modes:\n",
+    "\n",
+    "| Feature | Default Mode | Custom Mode |\n",
+    "|---------|-----------|-------------|\n",
+    "| Ideal For | Prototyping what's possible, focusing on dataset cleaning, finetuning, and evaluation pipeline | Optimizing model quality by controlling more parameters, hardware control |\n",
+    "| Command | `llmforge anyscale finetune config.yaml --default` | `llmforge anyscale finetune config.yaml` |\n",
+    "| Model Support | Popular models with their prompt format (e.g., `meta-llama/Meta-Llama-3-8B-Instruct`)* | Any HuggingFace model, any prompt format (e.g., `meta-llama/Meta-Llama-Guard-2-8B`) |\n",
+    "| Task Support | Instruction tuning for multi-turn chat | Causal language modeling, Instruction tuning, Classification|\n",
+    "| Data Format | Supports chat-style datasets, with fixed prompt formats per model | Supports chat-style datasets, with flexible prompt format |\n",
+    "| Hardware | Automatically selected (limited by availability) | User-configurable |\n",
+    "| Fine-tuning type| Only supports LoRA (Rank-8, all linear layers) | User-defined LoRA and Full-parameter |\n",
+    "\n",
+    "*NOTE: old models will get deprecated\n",
+    "\n",
+    "Choose the mode that best fits your project requirements and level of customization needed."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "# [Optional] You can set the WandB API key to track model performance\n",
-    "# import os\n",
-    "# os.environ[\"WANDB_API_KEY\"]=\"YOUR_WANDB_API_KEY\"\n",
+    "### Models Supported in Default Mode\n",
+    "\n",
+    "Default mode supports a select list of models, with a fixed cluster type of 8xA100-80G. For each model we only support context lengths of 512 up to Max. context length in increments of 2x (i.e. 512, 1024, ...). Here are the supported models and their configurations:\n",
```

Comment:
Nit: Can we call this list with a better name to reference everywhere? Like "Core Models" .